react/productSummary.css

```diff
@@ -5,7 +5,12 @@
 .imageContainer {}
 .information {}
 .element {}
-.image {}
+.imageInline {
+  max-height: 7rem;
```

Comment:
Is this really necessary? Couldn't the boundaries be set in a parent component using Tachyons?

Reply:
I'm afraid it is. It need to be set directly on the image for IE compatibility.